code/polygon-area/src/test/java/ru/unn/agile/polygon/model/LineTest.java

```diff
@@ -0,0 +1,66 @@
+package ru.unn.agile.polygon.model;
+
+import org.junit.Test;
+
+import static org.junit.Assert.*;
+
+public class LineTest {
+    private double delta = 0.0001;
+    private final Point p1 = new Point(0, 0);
+    private final Point p2 = new Point(1, 0);
+    private final Point p3 = new Point(1, 1);
+    private final Point p4 = new Point(0, 1);
+    private final Point p5 = new Point(5, 1);
+
+    @Test
+    public void canCreateLine() {
+        try {
+            Line line = new Line(p1, p2);
+        } catch (Exception e) {
+            fail("Should not have thrown any exception");
+        }
+    }
+
+    @Test
+    public void canGetLinePoints() {
+        Line line = new Line(p1, p2);
+
+        assertEquals(p1, line.getPoint1());
+        assertEquals(p2, line.getPoint2());
+    }
+
+
+    @Test
+    public void canGetLineCoordinates() {
+        Line line = new Line(p1, p2);
+
+        assertEquals(p2.getX() - p1.getX(), line.getX(), delta);
+        assertEquals(p2.getY() - p1.getY(), line.getY(), delta);
+
+        assertEquals(p1.getX(), line.getXPoint1(), delta);
+        assertEquals(p1.getY(), line.getYPoint1(), delta);
+
+        assertEquals(p2.getX(), line.getXPoint2(), delta);
+        assertEquals(p2.getY(), line.getYPoint2(), delta);
```

Comment:
Could you please use one assert per one test in all your tests?

Reply:
done.